base_layer/wallet/src/output_manager_service/master_key_manager.rs

```diff
@@ -142,6 +143,7 @@ where TBackend: OutputManagerBackend + 'static
 
     /// Return the next pair of (spending_key, script_private_key) from the key managers. These will always be generated
     /// in tandem and at corresponding increments
+    #[instrument(name = "key_manager::get_next_spend_and_script_key", skip(self))]
```

Comment:
I would not add any tracing to this file at the risk of leaking information

Reply:
Have removed them, thanks.